ui/src/app/views/cohort-edit/component.ts

```diff
@@ -15,6 +15,7 @@ export class CohortEditComponent implements OnInit {
   adding = false;
   workspaceNamespace: string;
   workspaceId: string;
+  buttonClicked = false;
```

Comment:
I just pushed a new version that adds a loading wheel to the button after it is clicked. (The widget library we were using has a really easy way to do that.) David Mohs and I think that may be sufficient as far as providing feedback goes.